app/models/mailing_list/wizard.rb

```diff
@@ -20,7 +20,7 @@ def complete!
         break unless result
 
         add_member_to_mailing_list
-        @store.purge!
+        @store.prune!(leave: %w[first_name last_name])
```

Comment:
Looks like `last_name` isn't referenced in the view?

Reply:
Yeah - I added that because it looked weird if you restart the wizard and your first name is pre-filled and your last name isn't!

Reply:
Ah that makes sense; worth adding a quick comment to that effect so someone doesn't accidentally refactor it out later?